### PR TITLE
Final extension for papers/notebooks - May 15

### PR DESCRIPTION
**Pull request by mrmundt**
## Description
Post the final extension for papers and notebooks.

## Checklist:

- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2023-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @mrmundt @cabejackson @manning-ncsa @jmelot @jbteves @jsubida

### CODE REVIEW

File: pages/participate/participate.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ the scientific community.
 **Timeline**
 
 - **Submissions open**: Tuesday, February 14, 2023
-- **Papers / Notebooks due**: Monday, May 8, 2023 - **EXTENDED**
+- **Papers / Notebooks due**: Monday, May 15, 2023 - **EXTENDED**
 - **Notification of authors for papers / notebooks**: Monday, June 12, 2023
```
Comment (jdamerow) [inline pages/participate/participate.md:76]: this would needed to be changed too, wouldn't it?

File: pages/participate/participate.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ about notebook submissions.
...
```
Comment (jdamerow) [inline pages/participate/participate.md:113]: this would needed to be changed too, wouldn't it?

Comment (mrmundt) [reply pages/participate/participate.md:113]: Changed to... What?

**Review by mchenry — APPROVED** [review]

File: pages/participate/participate.md (hunk 1 above)
Comment (mrmundt) [reply pages/participate/participate.md:76]: Oh oh oh, GOT IT. Misunderstood. Fixing!

**Pull request merged**